psalm-baseline.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="5.25.0@01a8eb06b9e9cc6cfb6a320bf9fb14331919d505">
+<files psalm-version="6.8.8@1361cd33008feb3ae2b4a93f1860e14e538ec8c2">
+  <file src="src/App/src/Handler/HomePageHandler.php">
+    <MissingOverrideAttribute>
```

Comment:
Can you put a suppression for `MissingOverrideAttribute` in `psalm.xml` and reduce the baseline?

```xml
    <issueHandlers>
        <MissingOverrideAttribute>
            <errorLevel type="suppress">
                <directory name="src" />
                <directory name="test" />
            </errorLevel>
        </MissingOverrideAttribute>
    </issueHandlers>
```

This makes it easier for users to opt-in if they want to and just run `psalter --issues=MissingOverrideAttribute` after removing the config

Reply:
@gsteel sure, updated https://github.com/mezzio/mezzio-skeleton/pull/175/commits/7c388eba5a826b3f62a23f0e7e7bee9bff37ee62